_posts/2018-10-21-nabla-k8s.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: post
+title: Nabla on Kubernetes!
+---
+
+
+A while back, we released a demo of the nabla containers runtime, `runnc` (v0.1). Most recently, we have improved our runtime in several aspects such as better start-up times and memory density. In addition to that (and also the focus of this blogpost), we have implemented Nabla support for kubernetes! In this blogpost, we will show a simple local setup of kubernetes using the nabla containers runtime, `runnc` (v0.2), through [containerd](https://github.com/containerd/containerd)  via the [CRI untrusted workload cri plugin and pod annotation](https://github.com/containerd/cri/blob/master/pkg/annotations/annotations.go).
+
+
+There will be several steps to achieve this on a local machine:
+
+1. Build and install Nabla runtime `runnc`
+2. Setup containerd with `runnc`
+3. Setup a local kubernetes cluster with containerd
+4. Setup a CNI config
+5. Running Nabla on k8s!
+
+We note for this post, the following versions were used:
+```
+runnc: v0.2
+containerd: v1.2.0-rc1
+kubernetes: v1.11.0
+```
+
+##  Build and install Nabla runtime `runnc`
+
+
+The first step to running nabla on kubernetes is to run nabla! The first order of business is to make sure we have a running new (v0.2) version of `runnc`. For a step-by-step guide to install `runnc`, take a look at our previous article: [Running a Nabla Container]({% post_url 2018-06-28-nabla-setup %}).
+
+**WARNING: Be sure to use latest commit!**
```

Comment:
Specify the version of runnc that you tested here (so these exact steps are valid forever and ever).